image-layout.md

```diff
@@ -2,6 +2,7 @@
 
 * The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 * This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
+* This section defines the `application/vnd.oci.image.layout.v1+json` [media type](media-types.md).
```

Comment:
I see what you mean, but would also argue that it's more appropriate in those other documents because there the bulk of the text is describing the type, whereas in this case it's only a small part of the document. But ¯\_(ツ)_/¯ , other maintainers can weigh in